lib/vet360/contact_information/email_update_response.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'vet360/response'
+require 'vet360/contact_information/async_response'
+
+module Vet360
+  module ContactInformation
+    class EmailUpdateResponse < Vet360::ContactInformation::AsyncResponse
+      attribute :email, Hash
+
+      def initialize(status, response = nil)
+        # @TODO parse and assign the :email data here?
```

Comment:
I would accept the original raw response here, set an attribute so we have a record of what the original response was, and construct the model based on it. Then set the attribute as part of the `super` call to the constructor. Check out `person_response.rb` for a similar pattern.